docs/src/main/sphinx/connector/hive-azure.rst

```diff
@@ -12,29 +12,24 @@ Trino supports both ADLS Gen1 and Gen2. With ADLS Gen2 now generally available,
 we recommend using ADLS Gen2. Learn more from `the official documentation
 <https://docs.microsoft.com/en-us/azure/data-lake-store/data-lake-store-overview>`_.
 
-Hive connector configuration
-----------------------------
-
-All configuration for the Azure storage driver is stored in the Hadoop
-``core-site.xml`` configuration file. The path to the file needs to be
-configured in the catalog properties file:
+Hive connector configuration for Azure Storage credentials
+----------------------------------------------------------
 
-.. code-block:: text
+All of the configuration properties for the Azure storage driver are stored in
+the Hadoop ``core-site.xml`` configuration file. To configure Trino to use the
+Azure Storage credentials, set the following configuration properties in the
+catalog properties file. It is best to use this type of configuration if the
+primary storage account is linked to the cluster.
 
-    hive.config.resources=<path_to_hadoop_core-site.xml>
+When there are secondary storage accounts involved, we recommend configuring
+Trino using a ``core-site.xml`` containing the appropriate credentials for each
+account, as described in the previous section.
 
-Configuration for Azure Storage credentials
--------------------------------------------
+The path to the file must be configured in the catalog properties file:
 
-If you do not want to rely on Hadoop's ``core-site.xml`` and want to have Trino
-configured independently with the storage credentials, you can use the following
-properties in the catalog configuration.
+.. code-block:: text
 
-We suggest to use this kind of configuration when you only have the Primary
-Storage account linked to the cluster. When there are secondary storage
-accounts involved, we recommend configuring Trino using a ``core-site.xml``
-containing the appropriate credentials for each account, as described in the
-preceding section.
+    hive.config.resources=<path_to_hadoop_core-site.xml>
```

Comment:
I haven't used so far `core-site.xml` to connect to abfs. I think it would be beneficial to have a very simplistic snippet out of a `core-site.xml` file listed below to give a feeling to a newbie user how this file looks like.

Reply:
I agree here, a minimal example would be great here.

Reply:
@mfzarko is there a specific example you have in mind that I can copy-paste?

Reply:
Sure, 
look here: https://hadoop.apache.org/docs/stable/hadoop-azure/abfs.html

```
The abfswales1.dfs.core.windows.net account is the name by which the storage account will be referred to.

Now ask for the connection string to the store, which contains the account key

az storage account  show-connection-string --name abfswales1
{
  "connectionString": "DefaultEndpointsProtocol=https;EndpointSuffix=core.windows.net;AccountName=abfswales1;AccountKey=ZGlkIHlvdSByZWFsbHkgdGhpbmsgSSB3YXMgZ29pbmcgdG8gcHV0IGEga2V5IGluIGhlcmU/IA=="
}
You then need to add the access key to your core-site.xml, JCEKs file or use your cluster management tool to set it the option fs.azure.account.key.STORAGE-ACCOUNT to this value.

<property>
  <name>fs.azure.account.key.abfswales1.dfs.core.windows.net</name>
  <value>ZGlkIHlvdSByZWFsbHkgdGhpbmsgSSB3YXMgZ29pbmcgdG8gcHV0IGEga2V5IGluIGhlcmU/IA==</value>
</property>

```